lambda/index.js

```diff
@@ -0,0 +1,148 @@
+const puppeteer = require('puppeteer-core');
```

Comment:
It would be nice to explore whether using `playwright` is an option these days. That would unify with our e2e pipelines and allow us to do a lot of interesting things.

Reply:
Yep definitely something to do for future.